src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/scanneddocs/ManageScannedDocsContestedAboutToSubmitHandler.java

```diff
@@ -147,4 +159,16 @@ private void setDefaultsForDocumentTypes(UploadCaseDocumentCollection document)
 
     }
 
+    private void removeProcessedScannedDocumentsFromCase(
+        FinremCaseData caseData, List<String> processedScannedDocumentIds) {
+        processedScannedDocumentIds.forEach(id -> removeScannedDocumentFromCase(caseData, id));
+    }
+
+    private void removeScannedDocumentFromCase(FinremCaseData caseData, String id) {
+        Optional<ScannedDocumentCollection> scannedDocument = caseData.getScannedDocuments().stream()
+            .filter(sdc -> sdc.getId().equals(id))
+            .findFirst();
```

Comment:
what if there is more than one? if it isn't possible than findAny

Reply:
Updated